batcher/aligned-batcher/src/lib.rs

```diff
@@ -886,9 +894,24 @@ impl Batcher {
             BatchQueueEntryPriority::new(replacement_max_fee, nonce),
         );
 
-        let updated_min_fee_in_batch = batch_state_lock.get_user_min_fee_in_batch(&addr);
+        // update max_fee_limit
+        let updated_max_fee_limit_in_batch = batch_state_lock.get_user_min_fee_in_batch(&addr);
         if batch_state_lock
-            .update_user_min_fee(&addr, updated_min_fee_in_batch)
+            .update_user_max_fee_limit(&addr, updated_max_fee_limit_in_batch)
+            .is_none()
+        {
+            std::mem::drop(batch_state_lock);
+            warn!("User state for address {addr:?} was not present in batcher user states, but it should be");
+            return;
+        };
```

Comment:
We should send a message to the client if this fails, as his connection won't stop otherwise.

Reply:
done